homeassistant/components/v2c/sensor.py

```diff
@@ -75,6 +75,21 @@ class V2CSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.POWER,
         value_fn=lambda evse_data: evse_data.fv_power,
     ),
+    V2CSensorEntityDescription(
+        key="slave_error",
+        translation_key="slave_error",
+        value_fn=lambda evse_data: evse_data.slave_error.name,
+        entity_registry_enabled_default=False,
+    ),
```

Comment:
What are the possible values?

Reply:
https://github.com/dgomes/pytrydan/blob/main/src/pytrydan/models/trydan.py#L23

Reply:
Can we make a transformation map in the integration that translates the enum to a str (snake_case) and then we can make this an enum sensor with the options provided. This allows the states to be translatable

Reply:
done! Thanks for a great suggestion :)